samza-core/src/main/scala/org/apache/samza/container/SamzaContainer.scala

```diff
@@ -418,12 +420,23 @@ object SamzaContainer extends Logging {
       .orNull
     info("Got checkpoint manager: %s" format checkpointManager)
 
+    val metadataStoreFactory = Option(config.getStartpointMetadataStoreFactory)
+      .map(Util.getObj(_, classOf[MetadataStoreFactory]))
+      .orNull
+    var startpointManager = None: Option[StartpointManager]
```

Comment:
Minor: Is the type needed? If so, could you put it after the variable?
e.g. "var startpointManager: Option[StartpointManager] = None"

Reply:
👍 Btw, the type should be Option, not None or Some.